qubesbuilder/plugins/installer/__init__.py

```diff
@@ -68,26 +68,36 @@ def __init__(
         self.iso_version = ""
         self.iso_timestamp = ""
         self.templates = templates
-        self.kickstart_path = Path(config.installer_kickstart)
 
+        self.kickstart_path = Path(config.installer_kickstart)
         if config.installer_kickstart.startswith("./"):
-            self.kickstart_path = self.kickstart_path.resolve()
-        if not (
-            self.manager.entities["installer"].directory / self.kickstart_path
-        ).exists():
-            raise InstallerError(
-                f"Cannot find kickstart: '{self.config.installer_kickstart}'"
+            self.kickstart_path = self.kickstart_path.relative_to(
+                self.config.get_conf_path().directory
             )
+        elif not config.installer_kickstart.startswith("/"):
+            self.kickstart_path = self.kickstart_path.relative_to(
+                self.get_sources_dir() / "qubes-release"
+            )
+        else:
+            self.kickstart_path = self.kickstart_path.expanduser().resolve()
+
+        if not self.kickstart_path.exists():
+            raise InstallerError(f"Cannot find kickstart: '{self.kickstart_path}'")
 
         self.comps_path = Path(config.installer_comps)
         if config.installer_comps.startswith("./"):
-            self.comps_path = self.comps_path.resolve()
-        if not (
-            self.manager.entities["installer"].directory / self.comps_path
-        ).exists():
-            raise InstallerError(
-                f"Cannot find kickstart: '{self.config.installer_comps}'"
+            self.comps_path = self.comps_path.relative_to(
+                self.config.get_conf_path().directory
+            )
+        elif not config.installer_comps.startswith("/"):
+            self.comps_path = self.comps_path.relative_to(
+                self.get_sources_dir() / "qubes-release"
```

Comment:
This doesn't work, I think `relative_to` does something. Should be maybe:
```suggestion
            self.comps_path = self.get_sources_dir() / "qubes-release" / self.comps_path
```
?
(and similar for kickstart)

Reply:
And also, later usage in `update_parameters` and in `run` might need a change too. It tries to copy_in into installer/conf (which might be okay?), but it will be problematic for kickstart that do include other files (that are not copied in anymore)

Reply:
For the include part, to be honest right now, I've no idea. If we need to resolve all the include that's going to be fun